src/App.vue

```diff
@@ -364,23 +331,23 @@ export default class App extends Vue {
   /** Data for fee summary component. */
   get feeFilingData (): Array<FilingDataIF> {
     let filingData = [] as Array<FilingDataIF>
+
     if (this.getFilingData) {
       filingData = cloneDeep(this.getFilingData)
-      if (this.isTypeCoop && this.getFilingData.length > 0) {
-        // Only set Future Effective and Priority to Special Resolution Fee
-        const specialResolutionFilingData = filingData.find(x => x.filingTypeCode === FilingCodes.SPECIAL_RESOLUTION)
-        if (specialResolutionFilingData) {
-          if (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE) {
-            filingData.forEach(x => {
-              x.waiveFees = true
-            })
-          } else {
-            specialResolutionFilingData.futureEffective = this.getEffectiveDateTime.isFutureEffective
-            specialResolutionFilingData.priority = this.getStaffPaymentStep.staffPayment.isPriority
-          }
+
+      if (this.isTypeCoop && this.isDissolutionFiling) {
```

Comment:
These look like the changes from a previous commit. I'm hoping a rebase makes these diffs go away.